gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/GfxdSerializable.java

```diff
@@ -367,4 +367,6 @@
   public final static byte LEAD_NODE_CONN_OP_MSG = 123;
 
   public final static byte LEAD_NODE_CONN_OP_CTX = 124;
+
+  public final static byte LEAD_NODE_GET_STATS = 125;
```

Comment:
As per convention please move a message type above to be < 75. This allows processorId to be set and any exceptions in deserialization to be sent back to source else it hangs.